overlays/holo-nixpkgs/hpos-holochain-api/src/index.js

```diff
@@ -39,25 +51,31 @@ const getPresentedHapps = async usageTimeInterval => {
         }
       }
       presentedHapps.push(happServiceloggerError)
-      break
     }
-
-    const { source_chain_count: sourceChains, bandwidth, cpu, disk_usage: storage } = appStats
-    usage.cpu = cpu
-    usage.bandwidth = bandwidth
-
-    presentedHapps.push({
-      id: happs[i].happ_id,
-      name: happs[i].happ_bundle.name,
-      enabled,
-      sourceChains,
-      usage,
-      storage
-    })
   }
   return presentedHapps
 }
 
+const registerECHapp = async url => {
+  const appWs = await AppWebsocket.connect(`ws://localhost:${HAPP_PORT}`)
+  const APP_ID = await getAppIds()
+  const ecHappBundle = {
+    hosted_url: "https://chat.holo.host",
```

Comment:
```suggestion
    hosted_url: "https://elemental-chat.holo.host",
```

Might as well use the real one :)